Classes/Notifications/NotificationsViewController.swift

```diff
@@ -47,53 +64,76 @@ FlatCacheListener {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        resetLeftBarItem()
-        if !showRead {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(
-                title: NSLocalizedString("Archives", comment: ""),
+        resetRightBarItem()
+
+        switch inboxType {
```

Comment:
`if case .unread = inboxType`? See my beautiful [blog post](https://basthomas.github.io/switches-and-cases).

Reply:
TIL!